db/mongo/client.go

```diff
@@ -2,6 +2,8 @@ package mongo
 
 import (
 	"context"
+	"github.com/ahab94/ApnaSchool/configuration"
```

Comment:
make sure to arrange imports properly.

1: golang packages

2: 3rd party

3: project packages